amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/jobs/main-indexing/job.json

```diff
@@ -6,7 +6,7 @@
     "nthreads": 45,
     "synapse_config": {
         "class": "File",
-        "path": "./.synapseConfig"
+        "path": "../../.synapseConfig"
```

Comment:
We need to come up with an absolute path strategy. I wish I could use "$HOME/.synapseConfig".

Reply:
i think the only way would be to auto generate json file on the fly from a config file.

Reply:
Agree. However, I think it will be simpler and more understandable to the end user of these workflows if we specify that they put the synapse config file in a standard location, perhaps somewhere under /etc.